heutagogy/persistence.py

```diff
@@ -27,14 +27,14 @@ def init_fn(db):
             db.cursor().executescript(f.read())
     with_connection(init_fn)
 
-def save_bookmark(bookmark):
+def save_bookmark(user_id, bookmark):
     def save_fn(conn):
         c = conn.cursor()
         c.execute('''
-            INSERT INTO bookmarks(timestamp, url, title, read)
-            VALUES (?, ?, ?, ?)
+            INSERT INTO bookmarks(user, timestamp, url, title, read)
```

Comment:
I am a noob in SQL but didn't you think about having the users in separate table?